src/utils/txma-header.ts

```diff
@@ -0,0 +1,9 @@
+import { Request } from "express";
+
+export function getTxmaHeader(req: Request): string {
```

Comment:
I know this is short enough but it could be simplified to
`req.headers["txma-audit-encoded"] as string ?? "";`